README.md

```diff
@@ -1,6 +1,25 @@
 # osbuild-composer
 
-An image building service based on
-[osbuild](https://github.com/osbuild/osbuild).  It is inspired by
-[lorax-composer](https://github.com/weldr/lorax) and exposes the same API. As
-such, it is a drop-in replacement.
+An HTTP service for building bootable OS images. It provides the same API as [lorax-composer](https://github.com/weldr/lorax) but in the background it uses [osbuild](https://github.com/osbuild/osbuild) to create the images.
+
+You can control it in [Cockpit](https://github.com/weldr/cockpit-composer) or using the [composer-cli](https://weldr.io/lorax/composer-cli.html). To get started on Fedora, run:
+
+```
+# dnf install cockpit-composer golang-github-osbuild-composer composer-cli
+# systemctl enable --now cockpit.socket
+# systemctl enable --now osbuild-composer.socket
+# systemctl enable --now osbuild-worker@1
```

Comment:
Starting the worker manually should be redundant right now.

Reply:
oh I didn't know, that's nice :)